tests/Sniffs/Operators/data/NegationOperatorSpacingSniffRequireSpaceErrors.fixed.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+$a = $a-$b;
+$a = $a  -  $b;
+$a =  - $a;
+$a =  self::CONST-  $a;
+$a = - (15 + 3);
+if ($a > - 16 || $a > - $b || $a > - self::CONSTANT) {
+
+}
+
+
+yield - 1;
```

Comment:
More detail in the main review comment, but this is an absolute blocker for me. The space after the `-` sign makes this look like a computation when we're yielding a literal value (`-1`). This definitely needs to be changed.

Reply:
In this case it's not about making this configurable or not - it's about `- 1` being significantly different from `-1`. `$a = 0 -5;` is incorrect since `-` is a subtraction operator in this context, but `$a = - 5;` is also incorrect because `-` is not a subtraction operator. Hence the issue with `yield - 1;`